hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -85,6 +85,12 @@ def pulpcore_enabled?
     module_enabled?('foreman_proxy_content')
   end
 
+  def local_db_exists?(database = 'foreman')
```

Comment:
I don't think we need a default
```suggestion
  def local_db_exists?(database)
```